detail.scss

```diff
@@ -0,0 +1,72 @@
+$background-color:#FFF;
+$border-color:#BED23C;
+$main-font:"Roboto";
+
+html { 
+  font-family: $main-font, sans-serif;
+}
+
+.title {
+  height: 39px;
+  width: 410px;
+  color: $equalizer-black;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 28px;
+  display:inline-flex;
+}
+
+.genre {
+  height: 30px;
+  width: 98px;
+  color: $equalizer-grey;
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 28px;
+  display:inline-flex;
+  padding-left: 8px;
+}
+
+.details {
+  margin-top: 50px;
+}
+
+.detail {
+  font-size: 20px;
+  display: flex;
+  margin-bottom: 36px;
+  color: $equalizer-grey;
+
+  strong {
+    color: $equalizer-black;
+    padding-right: 8px;
+  }
+}
+
+.image-cover {
+  margin :24px 70px 80px 30px;
+}
+
+.book-cover {
+  height: 425px;
+  width: 900px;
+  background-color: $background-color;
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
+  margin :15px 0 0 150px;
+}
+
+.badge {
+  height: 118px;
+  width: 84px;
+  transform: rotate(15deg);
+  position:absolute;
+  margin-left:246px;
```

Comment:
If you are using position absolute, place the element with top/bottom and right/left. Remember to avoid the use of margins to position elements 😄

Reply:
Done